stories/atoms/MultiSelectItem.stories.tsx

```diff
@@ -1,9 +1,16 @@
 import * as React from 'react';
 import type { Meta, StoryObj } from '@storybook/react';
-import { MultiSelectItem, Text } from '@textkernel/oneui';
+import {
+    MultiSelectItem,
+    DropdownRoot,
+    DropdownTrigger,
+    DropdownContent,
+    Button,
+    Text,
+} from '@textkernel/oneui';
 
 export default {
-    title: 'Atoms/List/MultiSelectItem',
+    title: 'Molecules/Dropdown/MultiSelectItem',
```

Comment:
I think the Dropdown, and all its sub-stories can now move to an Atom, as we have no dependency on other OneUI components, is that correct?